src/pages/gen2/start/quickstart/index.mdx

```diff
@@ -414,26 +373,41 @@ Let's update the return statement of the above component to also have a button f
   return (
     <main>
       <h1>Hello, Amplify 👋</h1>
+      <button onClick={async () => {
+        // create a new Todo with the following attributes
+        const { errors, data: newTodo } = await client.models.Todo.create({
+          // prompt the user to enter the title
+          content: window.prompt("title"),
+          done: false,
+          priority: 'medium'
+        })
+        console.log(errors, newTodo);
+      }}>Create </button>
+
       <ul>
         {todos.map((todo) => (
-          <li key={todo.id}>{todo.name}</li>
+          <li key={todo.id}>{todo.content}</li>
         ))}
-        <button onClick={() => {
-          // create a new Todo with the following attributes
-          const { errors, data: newTodo } = await client.models.Todo.create({
-            // prompt the user to enter the title
-            title: window.prompt("title"),
-            description: window.prompt("description"),
-            done: false,
-            priority: 'medium'
-          })
-        }}>Create </button>
       </ul>
     </main>
   );
 }
 ```
 
+Create a couple to dos, then refresh the page to see them. You can also subscribe to new todos in your `useEffect` to have them live reload on the page.
+
+```js
+useEffect(() => {
+  const sub = client.models.Todo.observeQuery()
+    .subscribe({
+      next: ({ items, isSynced }) => {
+        setTodos(items)
```

Comment:
```suggestion
        setTodos([...items])
```

I'm almost sure you have to do something like this because otherwise React's reference won't be refreshed for the state.